intelligent-appointment/demo-web-app/api/FSIConnect/index.js

```diff
@@ -1,15 +1,46 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
-
 const axios = require('axios')
 
 const endPoint = `${process.env.ENVIRONMENT_URL}/api/data/${process.env.ENVIRONMENT_VERSION}`
 
+let token = '';

```

Comment:
can you please remove this code block (just anything related to token and requestToken from this PR? I'll add the proper code in another PR.